virttest/qemu_devices/qcontainer.py

```diff
@@ -1083,7 +1083,11 @@ def pflash_handler(firmware_name, cmd):
                     img_path, img_name = os.path.split(img_info.get("filename"))
 
                     pflash_vars_name = "%s_%s_VARS.fd" % (self.vmname, img_name)
-                    pflash_vars_path = os.path.join(img_path, pflash_vars_name)
+                    if os.path.isabs(img_path):
```

Comment:
suggest change to ==>  if img_params["image_raw_device"] == "yes":

Reply:
thanks a lot, corrected.